src/app/external-cluster-wizard/steps/external-cluster/provider/aks/template.html

```diff
@@ -14,137 +14,157 @@
 limitations under the License.
 -->
 <form [formGroup]="form"
-      fxLayout="row"
+      [fxLayout]="isDialogView() ? 'column' : 'row'"
       fxLayoutGap="30px"
       fxLayout.md="column"
       fxLayoutGap.md="30px"
       fxLayout.sm="column"
       fxLayoutGap.sm="30px">
+  <div [fxFlex]="isDialogView() ? '100%' : '50%'"
+       fxLayout="column"
+       fxLayoutGap="8px">
```

Comment:
PLESE be careful with WIZARD/DIALOG checks:

I can SEE `Kubernetes Version` **TWICE** in Wizard MODE


<img width="1257" alt="b647208cc04d13c9ff85d8340fee4e70" src="https://user-images.githubusercontent.com/17727069/180873424-6e7b2e83-ce00-4caa-ac2f-2f2409b4b47d.png">